doc/lua_api.txt

```diff
@@ -93,25 +93,11 @@ Mods
 
 Mod load path
 -------------
-Generic:
+Paths are relative to the directories listed in the Paths section above.
```

Comment:
I feel that it's necessary to list out all possible paths for the sake of clarity